packages/components/src/components/Select/SelectMenu.tsx

```diff
@@ -52,7 +73,13 @@ export const SelectMenu = ({ highlightedIndex, getMenuProps, getItemProps, child
     return (
         <RadixPopover.Portal>
             <RadixPopover.Content onOpenAutoFocus={handleOnOpenAutoFocus} className={styles.portal}>
-                <ul className={styles.menu} {...getMenuProps()} ref={ref} data-test-id="fondue-select-menu">
+                <ul
+                    className={styles.menu}
+                    {...getMenuProps()}
+                    ref={ref}
+                    data-has-interacted={hasInteractedSinceOpening ? 'true' : 'false'}
```

Comment:
since `hasInteractedSinceOpening` is a boolean, why not
```suggestion
                    data-has-interacted={hasInteractedSinceOpening}
```

Reply:
data attributes need to be a string afaik? am i wrong?

Reply:
don't think so, if memory serves it ends up as a string on the DOM anyway